### PR TITLE
New scheme of object mapping for StorIOContentResolver

### LINKED ISSUE #274: Default Mapping and Resolvers
Setting `MapFunc` and `Resolvers` for each Operation is pretty annoying 

We can allow user to set default `MapFunc`'s and `Resolvers` for `StorIOSQLite` and `StorIOContentResolver` and allow to override them per Operation. 

@nikitin-da what do you think?

### PR DESCRIPTION
**Pull request by artem-zinnatullin**
Closes #274 

Finally!

@nikitin-da PTAL and thank you very much for great integration tests!

### CODE REVIEW

**Comment by artem-zinnatullin** [comment]
retest this please

File: storio-content-resolver/src/androidTest/java/com/pushtorefresh/storio/content_resolver/impl/TweetMeta.java [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+        }
+
+        @NonNull
+        @Override
+        public PutResult performPut(@NonNull StorIOContentResolver storIOContentResolver, @NonNull Tweet tweet) {
+            final PutResult putResult = super.performPut(storIOContentResolver, tweet);
+
+            if (putResult.wasInserted()) {
+                final Uri insertedUri = putResult.insertedUri();
+                tweet.setId(insertedUri != null ? ContentUris.parseId(insertedUri) : null);
```
Comment (nikitin-da) [inline storio-content-resolver/src/androidTest/java/com/pushtorefresh/storio/content_resolver/impl/TweetMeta.java]: What do you think, shell we encapsulate this parsing inside `putResult`?

**Comment by nikitin-da** [comment]
Awesome=)

File: storio-content-resolver/src/androidTest/java/com/pushtorefresh/storio/content_resolver/impl/TweetMeta.java (hunk 1 above)
Comment (artem-zinnatullin) [inline storio-content-resolver/src/androidTest/java/com/pushtorefresh/storio/content_resolver/impl/TweetMeta.java]: Hm, good question, may be yes, if there will be requests for that

**Pull request merged**